workflow/controller/workflowpod.go

```diff
@@ -381,12 +383,26 @@ func (woc *wfOperationCtx) createWorkflowPod(ctx context.Context, nodeName strin
 		pod.Spec = *patchedPodSpec
 	}
 
+	var x *entrypoint.Image
+
 	for i, c := range pod.Spec.Containers {
 		if c.Name != common.WaitContainerName {
 			// https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes
 			if len(c.Command) == 0 {
-				x, err := woc.controller.entrypoint.Lookup(ctx, c.Image, entrypoint.Options{
-					Namespace: woc.wf.Namespace, ServiceAccountName: woc.execWf.Spec.ServiceAccountName, ImagePullSecrets: woc.execWf.Spec.ImagePullSecrets,
+				err := wait.ExponentialBackoff(retry.DefaultRetry, func() (bool, error) {
+					var lookupErr error
+					x, lookupErr = woc.controller.entrypoint.Lookup(ctx, c.Image, entrypoint.Options{
+						Namespace:          woc.wf.Namespace,
+						ServiceAccountName: woc.execWf.Spec.ServiceAccountName,
+						ImagePullSecrets:   woc.execWf.Spec.ImagePullSecrets,
+					})
+					if lookupErr != nil {
+						if errorsutil.IsTransientErr(lookupErr) {
+							return false, nil
```

Comment:
Could you please give what is the return value of the woc.controller.entrypoint.Lookup function? Is it within the range of IsTransientErr?